### PR TITLE
NOISSUE - Use github action for showing OpenAPI spec with Swagger UI

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milosevic iva@blokovi.com

### What does this do?

- Use github action for deploying Swagger UI with github pages on custom domain (`api.mainflux.io`)
- All API specification files moved to one `/api` folder
- Fix semantic error in openAPI specification for Auth

### Which issue(s) does this PR fix/relate to?
No issue

### CODE REVIEW

**Review by manuio — DISMISSED** [review]
LGTM

**Review by darkodraskovic — DISMISSED** [review]
LGTM

**Review by drasko — DISMISSED** [review]
LGTM

**Review by dborovcanin — DISMISSED** [review]

**Review by dborovcanin — COMMENTED** [review]
Please check if API docs are referenced in README files for respective services and fix those references, as well.

**Review by manuio — DISMISSED** [review]
LGTM

**Comment by blokovi** [comment]
> Please check if API docs are referenced in README files for respective services and fix those references, as well.

Thanks @dusanb94 , done.

File: auth/README.md [hunk 1]
```diff
@@ -103,6 +103,6 @@ If `MF_EMAIL_TEMPLATE` doesn't point to any file service will function but passw
 ## Usage
 
 For more information about service capabilities and its usage, please check out
-the [API documentation](openapi.yml).
+the [API documentation](https://api.mainflux.io/?urls.primaryName=auth-openapi.yml).
```
Comment (manuio) [inline auth/README.md:106]: Why this url? It should be `api/auth-openapi.yml`, right?

**Review by manuio — CHANGES_REQUESTED** [review]

File: auth/README.md (hunk 1 above)
Comment (blokovi) [reply auth/README.md:106]: Yes @manuio , raw yml file is there. But on api.mainflux.io domain, it will be shown in SwaggerUI which is more user friendly

**Review by manuio — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**